_templates/base.html

```diff
@@ -75,7 +75,7 @@
     <script src="{% static '/static/js/jquery.cookie.js' %}"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js" integrity="sha256-Sk3nkD6mLTMOF0EOpNtsIry+s1CsaqQC1rVLTAy+0yc= sha512-K1qjQ+NcF2TYO/eI3M6v8EiNYZfA95pQumfvcVrTHtwQVDG+aHRqLi/ETn2uB+1JqwYqVG3LIvdm9lj6imS/pQ==" crossorigin="anonymous"></script>    {% block extra_js %}
     {% endblock %}
-    <script async src="{% static '/static/js/main.js' %}"></script>
+    <script async src="{% static '/static/js/dl.js' %}"></script>
```

Comment:
just add the tracking call inline. no need to include an additional javascript.
basic tracking is already included. all you need to do is to add 
{% block end_body %}
    <script>
      analytics.track('visit_download');
    </script>
{% endblock %}